src/Makefile.config

```diff
@@ -1,42 +1,37 @@
-FSTAR_HOME=..
-include $(FSTAR_HOME)/ulib/gmake/z3.mk    # This pins $(Z3) ...
-include $(FSTAR_HOME)/ulib/gmake/fstar.mk # and $(FSTAR) for all sub-make calls
-
-# --------------------------------------------------------------------
-BIN=../bin
-
-# --------------------------------------------------------------------
-# Configuration of some platform-specific tools; eventually we will want a configure script
-
-ifeq ($(OS),Windows_NT)
-  FSC     = fsc
-  MSBUILD = ./msbuild.bat
-  UNAME   = Windows_NT
-  FSTAR   = ../bin/fstar.exe
-  RUNTIME =
-else
-  FSC     = fsharpc
-  # If can't find msbuild, use xbuild, but throw a warning
-  MSBUILD = $(shell which msbuild || (echo '\n\n\033[0;31mWarning:\033[0m could not find "msbuild", trying (deprecated) "xbuild"\n\n'>&2; which xbuild))
-  UNAME   = $(shell uname -s)
-  FSTAR   = ../bin/fstar-any.sh
-  RUNTIME = mono
-endif
-
-CONFIGURATION?=Release
-
-MSBUILD := $(MSBUILD) /verbosity:minimal /p:Configuration=$(CONFIGURATION)
-DOS2UNIX=$(shell which dos2unix >/dev/null 2>&1 && echo dos2unix || echo true)
-
-ifeq ($(UNAME),Darwin)
-  HEAD=ghead
-  SED=gsed
-  FIND=gfind
-else
-  HEAD=head
-  SED=sed
-  FIND=find
-endif
-
-# --------------------------------------------------------------------
-
+FSTAR_HOME=..
+include $(FSTAR_HOME)/ulib/gmake/z3.mk    # This pins $(Z3) ...
+include $(FSTAR_HOME)/ulib/gmake/fstar.mk # and $(FSTAR) for all sub-make calls
+
+# --------------------------------------------------------------------
+BIN=../bin
+
+# --------------------------------------------------------------------
+# Configuration of some platform-specific tools; eventually we will want a configure script
+
+ifeq ($(OS),Windows_NT)
+  FSC     = fsc
+  MSBUILD = ./msbuild.bat
+  UNAME   = Windows_NT
+  FSTAR   = ../bin/fstar.exe
+  RUNTIME =
+else
+  FSC     = fsharpc
+  # If can't find msbuild, use xbuild, but throw a warning
+  MSBUILD = $(shell which msbuild || (echo '\n\n\033[0;31mWarning:\033[0m could not find "msbuild", trying (deprecated) "xbuild"\n\n'>&2; which xbuild))
+  UNAME   = $(shell uname -s)
+  FSTAR   = ../bin/fstar-any.sh
+  RUNTIME = mono
+endif
+
+CONFIGURATION?=Release
+
+MSBUILD := $(MSBUILD) /verbosity:minimal /p:Configuration=$(CONFIGURATION)
+DOS2UNIX=$(shell which dos2unix >/dev/null 2>&1 && echo dos2unix || echo true)
+
+# Use options compatible between BSD and GNU versions, on macOS and Linux
+HEAD=head
+SED=sed
+FIND=find
+
+# --------------------------------------------------------------------
+
```

Comment:
Can we get a cleaner diff here? Or is there really a need to change the line endings in this file?

Reply:
I thought the official policy was to use LF endings, not CRLF ones; I can revert if necessary.

Reply:
Okay, it seems that it's indeed the policy :)
https://github.com/FStarLang/FStar/blob/master/CONTRIBUTIONS.md#line-endings

For future larger diffs I would still suggest separating line ending changes from the real changes one needs to review.